workload/scripts/workload-install.ps1

```diff
@@ -36,7 +36,8 @@ function New-TemporaryDirectory {
 
 function Test-Directory([string]$TestDir) {
     if (-Not $(Test-Path "$TestDir")) {
-        Write-Error "No target directory '$TestDir'."
+        New-Item -ItemType Directory -Path $TestDir
+        Write-Host "Create target directory '$TestDir'."
```

Comment:
`Test-Directory` method's responsibility is to verify the existence and write permission of the given directory. 
How about just using this method like... 
```powershell
Test-Directory (Get-Item $ManifestDir).parent
New-Item -ItemType Directory -Path $ManifestDir -Force
Test-Directory $ManifestDir
.....
```

Reply:
I understood that the problem was that the create directory operation was run in a function named `test-directory`.
However, even if the function is separated, similar codes will be repeated. 
How about changing the name to `check-directory`?

Reply:
Ok, how about changing it to `Ensure-Directory`? and we don't need to use the recursive call because `New-Item` already can create directories recursively.
```powershell
function Ensure-Directory([string]$TestDir) {
    if (-Not $(Test-Path "$TestDir")) {
        try {
            New-Item -ItemType Directory -Path $TestDir -ErrorAction stop
            [io.file]::OpenWrite($(Join-Path -Path $TestDir -ChildPath ".test-write-access")).Close()
        catch [System.UnauthorizedAccessException] {
            Write-Error "No permission to install. Try run with administrator mode."
        }
    }
}
```

Reply:
ok thanks